src/views/List.jsx

```diff
@@ -4,14 +4,17 @@ export function List({ data }) {
 	return (
 		<>
 			<p>
-				Hello from the <code>/list</code> page!
+				Hello from the <code>/list</code> page! Our first test
 			</p>
 			<ul>
 				{/**
```

Comment:
You have implemented the TODO so you should remove it so as not to confuse future devs in here